shell/bashrc

```diff
@@ -319,7 +324,6 @@ function phpbrew ()
             unset PHPBREW_PATH
             eval `$BIN env`
             __phpbrew_set_path
-            echo "phpbrew is turned off."
```

Comment:
`phpbrew off` is called internally from `phpbrew each` after iteration in case if there was no active phpbrew-managed PHP version prior to running `each`. In this case, w/o having this line removed, the `each` command would output this message.

We could add something like a `--silent` flag to the `off` command, but personally, I feel it's better to not echo anything by default. If the command exits with the zero code, it means it's succeeded and it doesn't have to return anything else. For instance, if we execute `rm file.txt`, it doesn't output anything like "file.txt has been successfully removed".